src/ui/camera.js

```diff
@@ -1588,6 +1635,21 @@ class Camera extends Evented {
             delta > 180 ? -360 :
             delta < -180 ? 360 : 0;
     }
+
+    // emulates frame function for some transform
+    _emulate(frame: Function, duration: number, initialTransform: Transform) {
+        const frameRate = 15;
+        const numFrames = duration * frameRate / 1000;
```

Comment:
Nit: just to be sure there's a `1.0` transform at the end:

```suggestion
        const numFrames = Math.ceil(duration * frameRate / 1000);
```